src/cases/containers/case-home/case-home.component.ts

```diff
@@ -25,11 +24,15 @@ export class CaseHomeComponent implements OnInit, OnDestroy {
 
   public navigationSubscription: Subscription;
 
+  public showSpinner$: Observable<boolean>;
+
   constructor(
     private readonly alertService: AlertService,
     private readonly errorNotifierService: ErrorNotifierService,
     private readonly navigationNotifier: NavigationNotifierService,
     private readonly store: Store<fromFeature.State>,
+    private commonLibLoadingService: CommonLibLoadingService,
```

Comment:
I'll approve the merge for now but can these be changed to `readonly`, please?

Reply:
Done